src/rpcq-python.lisp

```diff
@@ -158,6 +158,7 @@ import sys
 
 from warnings import warn
 from rpcq._base import Message
+from rpcq.messages import ParameterSpec, PatchTarget
```

Comment:
Hmmm.... this creates a circular import by also adding `from rpcq.messages import ParameterSpec, PatchTarget` to `rpcq.messages`

Reply:
Another good catch. Seems that the better thing to do is to formally record the dependence of this submodule of messages with some other module, & then to have the autogenerator emit a corresponding block of `import`s in the header.